tests/cypress/integration/block-editor.test.js

```diff
@@ -8,16 +8,20 @@ describe('Test Autoshare for Twitter with Block Editor.', () => {
 		});
 	});
 
+	beforeEach(() => {
+		cy.login();
+	});
```

Comment:
What do you think about applying [this technique](https://github.com/10up/cypress-wp-utils/pull/46) to improve the testing performance?

Reply:
Yes, it would be great to apply it. but it's not merged yet.

Reply:
Ohh. you are asking to use that technique directly here on this plugin. Yes, let me do that. I was waiting for get it merged :D

Reply:
I think we just need to copy [these lines](https://github.com/10up/cypress-wp-utils/pull/46/files#diff-6bd80aef2afe0d5b3dcd8c0f268a82b3d0b197ef23fab87ee54145dd12efe09a) and update the tests.

Reply:
I did [the same ](https://github.com/10up/simple-podcasting/pull/162/files#diff-6bd80aef2afe0d5b3dcd8c0f268a82b3d0b197ef23fab87ee54145dd12efe09a)for simple-podcasting.